cypress/support/fragments/inventory/search/browseCallNumber.js

```diff
@@ -0,0 +1,35 @@
+import { Select, TextInput, including, Button, MultiColumnListCell, MultiColumnListRow } from '../../../../../interactors';
+
+const inventorySearch = TextInput({ id: 'input-inventory-search' });
+const browseButton = Button('Browse');
+const row4 = MultiColumnListRow({ index: 4 });
+const row5 = MultiColumnListRow({ index: 5 });
+
+export default {
+  select() {
```

Comment:
already exists, you can find function by name selectBrowseCallNumbers

Reply:
done